modules/OSSMailScanner/scanneractions/BindServiceContracts.php

```diff
@@ -17,14 +17,22 @@ public function process(OSSMail_Mail_Model $mail)
 			return $returnIds;
 		}
 
+		$accountnumbers = array();
 		$accounts = $mail->getActionResult('Accounts');
 		if (!empty($accounts)) {
+			$keys = array('BindAccounts', 'BindContacts', 'BindLeads', 'BindHelpDesk');
+			foreach($keys as $key) {
+				$accountnumbers = array_merge($accountnumbers, $accounts[$key]);
+			}
+		}
+
+		if (!empty($accountnumbers)) {
 			$db = PearDatabase::getInstance();
 
 			$query = 'SELECT servicecontractsid FROM vtiger_servicecontracts '
 				. 'INNER JOIN vtiger_crmentity ON vtiger_crmentity.crmid = vtiger_servicecontracts.servicecontractsid '
-				. 'WHERE vtiger_crmentity.deleted = 0 && sc_related_to IN (' . implode(',', $accounts) . ') && contract_status = ?';
-
+				. 'WHERE vtiger_crmentity.deleted = 0 && sc_related_to IN (' . implode(',', $accountnumbers) . ') && contract_status = ?';
+		// / migoi
```

Comment:
Do not comment here, just add your details to  https://github.com/YetiForceCompany/YetiForceCRM/pull/4135/files#diff-a904325f94a346209085786124af0cd0R7
ex. https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/vendor/yetiforce/Privilege.php#L9